### ISSUE TITLE
Get Clearview Source Experience Built For Testing

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-ux#77]
_Issue within Epic #66_ 

---

## Problem
We need a version of SecureDrop to test with proxy source users via remote tooling such as usertesting.com or userfeel.com. To narrow the scope of the sessions we want to eliminate Tor dependencies to get users to hone-in on Source UI particulars, so some slight tweeks will need to be done to the Source UI messaging around Tor, and the instance will need to exist behind a normal URL.

## Solution
- Nina will slowpoke HTML/CSS, or Erik will cheerfully pick-up the ball? 😸 
  - Dependent on finishing #76 first
  - Nina will try finding a URL as well, to use for ongoing Source UI testing stuff

**Comment by ninavizz** [comment]
Nina purchased nyworld.org domain (with iwantmyname.com). 

The`.index` for that domain should point to FPF's main site, but the extended URLs we create for testing should live within the nyworld.org domain. Pls to coordinate with me for server stuff once ready, and I'll plug those values into the page with the DNS.

**Comment by eloquence** [comment]
I've successfully spun up the dev env through an nginx reverse proxy on nyworld.org (with HTTPS cert). Spun down again now; configuring it for the purposes of the test should be pretty straightforward.

**Comment by eloquence** [comment]
OK, I've configured it as follows:

- https://securedrop.nyworld.org/ - the source interface
- https://www.nyworld.org/ - any main page content we want
- https://www.nyworld.org/tips/ - any landing page content we want

(Server is down again now for security reasons; we'll just run it during the tests.)

I can easily change the Source Interface logo or other aspects of the configuration. For now I've just disabled the Tor-related banners, which don't make sense in a clearnet browser. Next steps to close this issue blocked on #76.
